python/ppml/src/bigdl/ppml/fl/nn/pytorch/estimator.py

```diff
@@ -36,7 +35,13 @@ def __init__(self, model: nn.Module, loss_fn, optimizer: torch.optim.Optimizer,
         self.fl_client = FLClient(target)
         self.loss_history = []
 
-    
+    @staticmethod
+    def from_torch(model: nn.Module,
```

Comment:
make it `ppml.vfl.Estimator.from_torch` instead

Reply:
do we need to make `ppml.fl....` and `ppml.vfl` co-exist or `ppml.fl.vfl...`

Reply:
Maybe `ppml.fl.Estimator` (if we can extend to HFL later)?

Reply:
@jason-dai `fl.Estimator` is available now. And note that `PytorchEstimator` could not extend `fl.Estimator` but have to extend another abstract class because of the circular dependency import issue. This is the same with the implementation in Orca